src/widgets/picture_widget.rs

```diff
@@ -50,10 +50,7 @@ pub enum MovementDir {
 
 impl MovementDir {
 	fn moving(self) -> bool {
-		match self {
-			MovementDir::None => false,
-			_ => true,
-		}
+		!matches!(self, MovementDir::None)
```

Comment:
You commented in the PR that you are not happy with `!matches!`. Yeah it's a bit odd but I think it's alright.

Reply:
Your call. I think it's a bit suboptimal because the `!` can be a bit hard to see but I agree that it's still alright.